outlines/1.3-migration

```diff
@@ -0,0 +1,31 @@
+1. Breaking changes (as we'd normally do this)
+
+2. Recommended changes : Modules [link to app structure guide]
+  1. Applications: Move all files to imports, initialize in one spot, link via `import`
+  2. Import *all* symbols from packages, including meteor core packages.
```

Comment:
This means we need to document where these symbols come from, which is currently completely not in the docs. It could also get a bit awkward, like:

```js
import { Meteor } from 'meteor';

// lol
Meteor.loginWithPassword
```